### ISSUE TITLE
Add an entity for spaces to status updates

### ISSUE THREAD

**Issue by jgmac1106** [idno/known#2695]
## While trying to do this:
Post a poem
## I encountered this error:

All my blank spaces get stripped in status updates thus changing the meaning of the poem

## Some other notes:
If I hit the space bar the space entity should not be sanitized or should be included. 
Give us some context: 

*

**Comment by Lewiscowles1986** [comment]
`&nbsp;` is a way around for HTML, which as a technology is generally not great with whitespace.

I believe the component you may be fighting here is either HTML or TinyMCE (graphical editor).

I wonder if known MCE strips or supports `<pre>` pre-formatted text. It is an option that it supports

**Comment by kevinmarks** [comment]
Another option is to use a utf-8 no-break space U+00A0 which you can type on a mac with option-space (I think that's the same on windows).
That tends to make it through utf8 chains OK
    indenting
      with these

**Comment by jgmac1106** [comment]
Thank you both!! I did not know. @Lewiscowles1986 status updates need to be stripped of their HTML stripping. It was set rigidly for Twitter, but who cares bout the silos. Messes with many a micropub client

@kevinmarks thank you!! that will work. I did not know.  

Going to keep issue open as it should just work without special characters

**Comment by Lewiscowles1986** [comment]
Careful not to do it in CSS... I think that is what caused a bug I reported to firefox and chrome teams a few years back

**Comment by benwerd** [comment]
I agree that status updates should allow HTML and not be subject to Twitter's restrictions.
